prog/postgres/postgres_resource_nexus.rb

```diff
@@ -134,21 +134,25 @@ def before_run
   end
 
   label def create_billing_record
-    BillingRecord.create_with_id(
-      project_id: postgres_resource.project_id,
-      resource_id: postgres_resource.id,
-      resource_name: postgres_resource.name,
-      billing_rate_id: BillingRate.from_resource_properties("PostgresCores", "standard", postgres_resource.location)["id"],
-      amount: representative_server.vm.cores
-    )
-
-    BillingRecord.create_with_id(
-      project_id: postgres_resource.project_id,
-      resource_id: postgres_resource.id,
-      resource_name: postgres_resource.name,
-      billing_rate_id: BillingRate.from_resource_properties("PostgresStorage", "standard", postgres_resource.location)["id"],
-      amount: postgres_resource.target_storage_size_gib
-    )
+    (postgres_resource.required_standby_count + 1).times do |index|
+      resource_type = (index == 0) ? "PostgresCores" : "PostgresStandbyCores"
+      BillingRecord.create_with_id(
+        project_id: postgres_resource.project_id,
+        resource_id: postgres_resource.id,
+        resource_name: postgres_resource.name,
+        billing_rate_id: BillingRate.from_resource_properties(resource_type, "standard", postgres_resource.location)["id"],
+        amount: representative_server.vm.cores
+      )
+
+      resource_type = (index == 0) ? "PostgresStorage" : "PostgresStandbyStorage"
+      BillingRecord.create_with_id(
+        project_id: postgres_resource.project_id,
+        resource_id: postgres_resource.id,
+        resource_name: postgres_resource.name,
+        billing_rate_id: BillingRate.from_resource_properties(resource_type, "standard", postgres_resource.location)["id"],
+        amount: postgres_resource.target_storage_size_gib
+      )
+    end
```

Comment:
```suggestion
    (postgres_resource.required_standby_count + 1).times do |index|
       ["Cores", "Storage"].each do |type|
         resource_type = (index == 0) ? "Postgres#{type}" : "PostgresStandby#{type}"
         BillingRecord.create_with_id(
           project_id: postgres_resource.project_id,
           resource_id: postgres_resource.id,
           resource_name: postgres_resource.name,
           billing_rate_id: BillingRate.from_resource_properties(resource_type, "standard", postgres_resource.location)["id"],
           amount: representative_server.vm.cores
         )
       end
     end
```

Reply:
I actually prefer the current version as it is more readable. I love shaving off some lines, but looping through array of array and generating resource_type using string concatenation seemed bit too much to me.

Reply:
There's a lot of common subexpression in your version though, is there a version without sub-string sp;licing that doesn't repeat, like, 

```
        project_id: postgres_resource.project_id,
        resource_id: postgres_resource.id,
        resource_name: postgres_resource.name,
```

among other content?

Reading the code to confirm that these are actually held in common is a bit of an eye-maze.

Reply:
I still prefer the old one, but I wrote a version that does not duplicate BillingRecord.create_with_id calls. How about this?

Reply:
I am OKish for the new version. To be honest, I personally prefer having the array written out as I have provided in the suggestion, but this is a personal preference. I leave the decision to you and @fdr